### DISCUSSION TITLE
Rewind your activities in the environment

### DISCUSSION THREAD

**Discussion by lalawuu** [tensorchord/envd#1137] [Ideas]
I saw a product fund by A16Z today. https://www.rewind.ai/

Could we do something fancy like this which can help users recall the develop process?

**Comment by gaocegege** [comment]
If you use envd-server, every ssh connection can be replayed with asciinema.org. https://asciinema.org/#

**Reply by lalawuu** [reply]
Maybe we can do more such as add some search function
